Formula/libusb.rb

```diff
@@ -37,8 +41,8 @@ def install
   test do
     cp_r (pkgshare/"examples"), testpath
     cd "examples" do
-      system ENV.cc, "-lusb-1.0", "-L#{lib}", "-I#{include}/libusb-1.0",
-             "listdevs.c", "-o", "test"
+      system ENV.cc, "listdevs.c", "-L#{lib}", "-I#{include}/libusb-1.0",
+             "-lusb-1.0", "-o", "test"
```

Comment:
Just a thought: wonder if we should be installing unversioned symlinks of `include/libusb-1.0` and `libusb-1.0.dylib` here.